ts/__tests__/Iterable.spec.ts

```diff
@@ -130,13 +129,13 @@ test("trackPurchase_params_methodCalled", () => {
   expect(MockRNIterableAPI.trackPurchase).toBeCalledWith(total, items, dataFields)
 })
 
-test("trackPurchase_paramsWithOptionalFields_methodCalled", () => {
-  Iterable.logger.log("trackPurchase_paramsWithOptionalFields_methodCalled")
+test('trackPurchase_paramsWithOptionalFields_methodCalled', () => {
+  Iterable.logger.log('trackPurchase_paramsWithOptionalFields_methodCalled')
 
   // GIVEN the following parameters
   const total = 5
-  const items = [new IterableCommerceItem("id", "swordfish", 64, 1, "SKU", "description", "url", "imageUrl", ["sword", "shield"])]
-  const dataFields = {"key": "value"}
+  const items = [new IterableCommerceItem('id', 'swordfish', 64, 1, 'SKU', 'description', 'url', 'imageUrl', ['sword', 'shield'])]
+  const dataFields = { key: 'value' }
```

Comment:
key doesnt need " or '?

Reply:
The linting is following this rule [here](https://typescript-eslint.io/rules/quotes/). We can turn off the rule if necessary.

Reply:
This one is good actually...